.github/CODEOWNERS

```diff
@@ -23,7 +23,7 @@
 /dolphinscheduler-registry/ @caishunfeng @ruanwenjun
 /dolphinscheduler-api/ @caishunfeng @SbloodyS
 /dolphinscheduler-dao/ @caishunfeng @SbloodyS
-/dolphinscheduler-dao/src/main/resources/sql/ @zhongjiajie
+/dolphinscheduler-dao/src/main/resources/sql/
```

Comment:
did some of our team interested be new code owner for `/dolphinscheduler-dao/src/main/resources/sql/` module?

Reply:
how about @EricGao888 ?

Reply:
Sure

Reply:
will add one more commit and assign those to you 💯